src/lib/ext-monitor.js

```diff
@@ -102,6 +98,47 @@ export default class ExtensionMonitor {
     return logs;
   }
 
+  async saveLogs({ blob, filename }) {
+    const url = URL.createObjectURL(blob);
```

Comment:
Aren't the log that we want to save already available in the `logs` property of the `ExtensionMonitor` class?

Sending a blob from the activitylog page seems unnecessary as it would send across IPC something that we do already have locally (and which can be potentially become pretty big if the monitoring session is long enough).

Even the filename doesn't seem to be necessary, given that the caller is sending a literal string value for it.

I think that this event handler could be written without taking any parameter for now (and we may add some, e.g. one with the filter options, once we actually need it).

Reply:
Updated it with empty parameter and using the logs from background.